content/en/security/application_security/api-inventory/_index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: API Security Inventory
+kind: documentation
+---
+
+## Overview
+
+API Security Inventory monitors your API traffic to provide visibility into the security posture of your APIs, including:
+
+- **Authentication**: type of authentication used (e.g. Basic Auth, API key, etc.)
+- **Public Exposure**: Whether the API is processing traffic from the internet
+- **Production status**: if the API is running in production environment
+- **Sensitive data flows**: Sensitive data handled by the API and flows between APIs.
+- **Attack Exposure**: if the endpoint is targeted by attacks (powered by Application Threat Management)
+- **Vulnerabilities**:  if the endpoint contains vulnerability (powered by Application Vulnerability Management)
+
+Using the API Security Inventory you can:
+
+- see at a glance your publicly exposed endpoints, which are not requiring authenticated requests
+- see at a glance your publicly exposed endpoints, which are handling sensitive data, such as payment card numbers
+- see which endpoints are at risk, and pivot directly into the [Threat Monitoring and Protection](/security/application_security/threats/) service for further investigation or response
+
+{{< img src="security/application_security/api/welcome.png" alt="API Security Inventory main page">}}
+
+## How to Enable
+
+### Via Remote Configuration
+
+If your library is configured to use [remote configuration][1], then the following tracer versions
```

Comment:
Ok, but how do they enable it? Are we enabling it via RC by default? Otherwise we can link to the place in the UI where they can enable it.

Reply:
@Julio-Guerra They'll be prompted to click on a button when they first go to the API page. 

We'll store a 10% sampling rate in RC at that point (and it'll unlock the UI). The sampling rate is not exposed to users though, for them it's presented as an "activate" button.